extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -79,13 +80,14 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 		$themWikiWrapper = $('#ThemeWiki');
 		steps = wb.find('.steps .step');
 		wikiName = $nameWikiWrapper.find('input[name=wiki-name]');
+		wikiNameLabel = $nameWikiWrapper.find('label[for=wiki-name]');
 		wikiNameStatus = $nameWikiWrapper.find('.wiki-name-status-icon');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "wikiNameStatus" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)